configs/ASRock/J5005-ITX.conf

```diff
@@ -0,0 +1,86 @@
+# libsensors configuration file
+# -----------------------------
+#
+# Motherboard: Asrock J5005-ITX
+# DMI: To Be Filled By O.E.M. To Be Filled By O.E.M./J5005-ITX, BIOS P1.40 08/06/2018
+# SuperIO chip: NCT6796D
+chip "nct6796-*"
+
+	set beep_enable 1
+
+	# Voltages:
+	label	in0	"Vcore"		# CPUVCORE
+	set	in0_min	0.16
+	set	in0_max	1.46
+	set	in0_beep 1
+
+	label	in1	"+12V"		# VIN1
+	compute	in1	@*(53/8), @/(53/8)
```

Comment:
These are not valid values. Most likely, the divider is made on resistors `R1 = 56 KΩ` and `R2 = 10 KΩ`. So it should be `66/10`.
Voltages read from UEFI and ASRock A-Tuning: `11.668, 11.721, 11.774, 11.827, 11.880, 11.932, 11.985, 12.038, 12.091, 12.144, 12.196, 12.249, 12.302, (12.355), 12.408, 12.460, (12.513), (12.725), (12.778)`
Values read in lm_sensors (`sensors -u nct6796-isa-0290 | grep "in1_input"`): `1.872, 1.896, 1.904, 1.912, 1.920, 1.928, 1.936`
Calculated voltages:
1.872 * 6.6 = 12.3552
1.896 * 6.6 = 12.5136
1.928 * 6.6 = 12.7248
1.936 * 6.6 = 12.7776